tutorials/admin/getting-started/updating.md

```diff
@@ -0,0 +1,64 @@
+# Updating the Software
+
+Occasionally Marble will release updates to its software. These updates may include bugfixes, feature updates, and
+security patches. 
+
+It is important to keep your Marble node up to date. This is especially important if your node is part of the Marble
+network so that we can ensure that all nodes in the network are secure and provide predictably similar features.
+
+### Updating to the latest version
+
+First navigate to the directory that contains the `birdhouse-deploy` source code:
+
+```shell
+cd birdhouse-deploy
+```
+
+Then use the `pavics-compose.sh` script to stop the stack momentarily while you update the code:
+
+```shell
+cd birdhouse
+./pavics-compose.sh down
+```
+
+Have a look at the [CHANGES.md](https://github.com/bird-house/birdhouse-deploy/blob/master/CHANGES.md) file to see what
```

Comment:
Include a line telling the user to go to the birdhouse repo.

Reply:
That's where that link goes. Maybe I don't understand what you're suggesting

Reply:
The flow of the text goes from telling the user to go to their local install folder to shut down birdhouse and then tells them to check CHANGE.md.  It's not clear that the file is on the remote repo.

Just adding the location to the sentence "Have a look at the Changes.md on the birdhouse repo." makes it more clear where to go.  The link is still kept.

Reply:
I see... ok sure